Formula/aerc.rb

```diff
@@ -1,8 +1,8 @@
 class Aerc < Formula
   desc "Email client that runs in your terminal"
   homepage "https://aerc-mail.org/"
-  url "https://git.sr.ht/~sircmpwn/aerc/archive/0.5.2.tar.gz"
-  sha256 "dec6560c1359d1d56124a85692e877e319036f0312ce9b7a31f9828f99b92c61"
+  url "https://git.sr.ht/~rjarry/aerc/archive/0.7.1.tar.gz"
+  sha256 "e149236623c103c8526b1f872b4e630e67f15be98ac604c0ea0186054dbef0cc"
```

Comment:
Why is this changing upstream?

Reply:
The original maintainer (@ddevault) has stopped working on aerc, and has suggested we use this fork: <https://git.sr.ht/~sircmpwn/aerc/commit/8d37356389>.

Currently https://aerc-mail.org/ still points towards Drew's original version of aerc2, but if you see the above commit is only about 12 hours old, he probably hasn't gotten around to updating the website. I'm going to send a patch now so it can be updated.